### PR TITLE
Rebuild for r-base 4.4 and UCRT mingw toolchain

### PR DESCRIPTION
**Pull request by danielnachun**

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

File: recipe/build.sh
Comment (mfansler) [inline recipe/build.sh]: We should always pass the arguments:

```suggestion
${R} CMD INSTALL --build . ${R_ARGS}
```

File: recipe/meta.yaml
Comment (mfansler) [inline recipe/meta.yaml]: Please don't undo this. That is the current boilerplate and should be retained.

```suggestion
    - $R -e "library('venneuler')"           # [not win]
    - "\"%R%\" -e \"library('venneuler')\""  # [win]
```

File: recipe/meta.yaml
Comment (mfansler) [inline recipe/meta.yaml:42]: Please restore.

File: recipe/meta.yaml
Comment (mfansler) [inline recipe/meta.yaml:2]: Retain.

File: recipe/meta.yaml
Comment (mfansler) [inline recipe/meta.yaml]: Retain.

**Review by mfansler — CHANGES_REQUESTED** [review]

**Comment by mfansler** [comment]
Hi @danielnachun. Since you're joining the team, maybe let's be a bit more strict about keeping the standard that would come out of `conda_r_skeleton_helper` rather than cutting off what might currently seem as unnecessary. The advantage to keeping the Windows boilerplate around is that all our recipes are robust to readily convertible to non-`noarch`.

Also, in the future please always commit your changes first, then run a `conda-smithy rerender` and commit with the suggested commit message. That provides a transparent record of when rerendering was done and the exact version of `conda-forge-pinning` that was applied.

**Comment by danielnachun** [comment]
Thanks @mfansler, I wasn't sure if those `#[win]` lines were leftovers that are supposed to be removed for `noarch` packages since I was mostly fixing non-`noarch` packages until now.  I guess for new R recipe submission in conda-forge they might result in linting failures, but for existing ones it's obviously allowed.  I'll also fix up my commit messages for the rerenders.  I guess I can use the bot to rerender most of the time after commit my manual changes as I've seen you do before.  I'll make sure to double check with other R maintainers before merging anything I'm not sure about.

**Comment by danielnachun** [comment]
I resolved all the requested changes here and made the same changes to the other PRs I made for license issues as well, so I'm going to merge them now.

**Pull request merged**
